assets/prefabs/Equipment/Silver/SilverReaper.prefab

```diff
@@ -0,0 +1,23 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:Reaper",
```

Comment:
Should be `"icon" : "Equipment:SilverReaper"`

This refers to where the icon is located, and the name of your image file is `SilverReaper.png`.